src/codeflare_sdk/cluster/cluster.py

```diff
@@ -217,27 +218,38 @@ def status(
 
         return status, ready
 
+    def is_dashboard_ready(self) -> bool:
+        try:
+            response = requests.get(self.cluster_dashboard_uri(), timeout=5)
+            if response.status_code == 200:
+                return True
+        except requests.RequestException:
+            pass
+        return False
+
     def wait_ready(self, timeout: Optional[int] = None):
         """
         Waits for requested cluster to be ready, up to an optional timeout (s).
         Checks every five seconds.
         """
         print("Waiting for requested resources to be set up...")
         ready = False
+        dashboard_ready = False
         status = None
         time = 0
-        while not ready:
+        while not ready or not dashboard_ready:
             status, ready = self.status(print_to_console=False)
+            dashboard_ready = self.is_dashboard_ready()
             if status == CodeFlareClusterStatus.UNKNOWN:
                 print(
                     "WARNING: Current cluster status is unknown, have you run cluster.up yet?"
                 )
-            if not ready:
+            if not ready or not dashboard_ready:
                 if timeout and time >= timeout:
                     raise TimeoutError(f"wait() timed out after waiting {timeout}s")
                 sleep(5)
                 time += 5
-        print("Requested cluster up and running!")
+        print("Requested cluster dashboard are up and running!")
```

Comment:
```suggestion
        print("Requested cluster and dashboard are up and running!")
```
Small nit pick

Reply:
Nice catch, thanks Mark! That's committed now.